sw/device/lib/base/log.c

```diff
@@ -64,11 +80,29 @@ void base_log_internal_core(log_severity_t severity, const char *file_name,
  * Logs `format` and the values that following in an efficient, DV-testbench
  * specific way.
  *
- * @param severity the log severity.
- * @param format a format string, as described in print.h. This must be a string
- * literal.
+ * @param log the log_fields_t struct that holds the log fields.
+ * @param nargs the number of arguments passed to the format string.
  * @param ... format parameters matching the format string.
  */
-void base_log_internal_dv(log_severity_t severity, const char *format, ...) {
-  // Do nothing, for now.
+
+/**
+ * Indicates the fixed location in RAM for SW logging for DV.
+ * TODO: Figure aout a better place to put this.
+ */
+static const uintptr_t kSwLogDvAddr = 0x1000fffc;
+
+void base_log_internal_dv(const log_fields_t *log, int nargs, ...) {
+  va_list args;
+  va_start(args, nargs);
+
+  // The first arg in the va_list is the format string - discard it.
+  va_arg(args, uint32_t);
```

Comment:
The comment got lost somewhere... you can discard this; just don't pass `_format` into the function, since it's not use.